#857-Stepper registration form

Apostall107 opened the pull request:
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/EventsExpress/EventsExpress/projects/2)


## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

Created stepper that is needed for Epic11 tasks.

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

Apostall107 committed:
Created reusable stepper component.

iuriikhrystiuk commented on EventsExpress/ClientApp/src/components/stepper/Stepper.js:
```diff
@@ -0,0 +1,149 @@
+export default class Stepper extends Component 
+{
+    constructor() 
+    {
```
We have a style, when in JS/CSS files new block symbols should be displayed on the same line of an element.

Apostall107 committed:
Updated stepper JS/CSS code style.

Apostall107 committed:
Visual update for event express font styles.

Apostall107 committed:
Added index js for stepper.

Apostall107 committed:
Success page added.
Page that notice user that process of registration completed.
Page has 2 redirect button to choose of:
- to look for events.
- to go to profile.

Apostall107 committed:
Added second step for stepper registration form.

Apostall107 committed:
Added new route for registration form.

Apostall107 committed:
Renamed AdditionalInfoForm to CompleteProfileForm for more clear understanding.

Apostall107 committed:
Added confirmation form and success form with check mark animation.

Apostall107 committed:
Fixed formating style of stepper files.

Apostall107 committed:
Demo view of the form created.

Apostall107 committed:
Namings of the steps changed.

Apostall107 committed:
Demo view of form created.

Apostall107 committed:
Added gendersArray for correct display of gender to user by naming, not by index of item.

Apostall107 committed:
Added date picker.
Now gender picker works correctly.

Apostall107 committed:
Some formating changes.

Apostall107 committed:
Code smells fixed

Apostall107 committed:
CodeSmell fix.
